### ISSUE TITLE
Conversion with convert results in strange output

### ISSUE THREAD

**Issue by paazmaya** [mgreter/sass2scss#20]
While building tests for https://github.com/paazmaya/grunt-sass2scss, came across strange behaviour.

The given Sass file:

```sass
/**
 * This is third file
*/
@import 'styles-2'

// Imports include the other file

label
  // Use the variable from the other file
  color: $color-white
```
with `sass2scss -c < input.sass > output.scss`, the resulted SCSS file:

```scss
/*** This is third file */*/ {}@import 'styles-2';/* Imports include the other file */label { /* Use the variable from the other file */color: $color-white; }
```

**Comment by mgreter** [comment]
You can actually shorten this to:

```
/**
*/
```

Have you tried compiling this with ruby sass? Because it does not! Somehow ruby sass expects the multiline comment closer to be indented. And my tests indicate that sass2scss mirrors that behavior pretty much perfectly. Unfortunately the sass specs are not very clear on such issues ...

**Comment by mgreter** [comment]
Just retried on sassmeister and the given code does still not compile.
